appveyor.yml

```diff
@@ -65,5 +65,5 @@ test_script:
  - echo %DC%
  - echo %PATH%
  - '%DC% --version'
- - make -f win32.mak rdmd dustmite ddemangle changed
- - make -f win32.mak test
+ - make -f win32.mak DFLAGS=-g rdmd dustmite ddemangle changed
+ - make -f win32.mak DFLAGS=-g test
```

Comment:
Is there anything we can do to add other test compilers for rdmd on Windows?  I didn't do this myself because I'm not very familiar with the appveyor setup (or D stuff on Windows in general).

Reply:
Doing it generically in `posix.mak` is pretty difficult, but if AppVeyor is the environment we want to support, then it should be much simpler. It's possible to just download a prebuilt LDC/GDC archive, unpack it, and use it. Targeting AppVeyor is nice even without access to a Windows machine - the CI has been pretty responsive for me, and they even remote access where you can RDP in and debug / experiment with the setup.